iphone/Classes/TiUIListView.m

```diff
@@ -1815,7 +1815,32 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
         [self.proxy fireEvent:@"dragstart" withObject:nil withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
     }
 }
+-(void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset{
+    
+    if ([self.proxy _hasListeners:@"scrolling"]) {
+        NSString* directionString = @"none";
+
+        if (velocity.y > 0) {
+            directionString = @"up";
+        }
 
+        if (velocity.y < 0) {
+            directionString = @"down";
+        }
+        
+        if(directionString != @"none") {
```

Comment:
I know, but just checking for nil was still not working. Please remember I'm no Objective C dev